src/scene.cpp

```
@@ -26,6 +26,13 @@
 #include "audio.h"
 #include "transition.h"
 
+#ifndef NDEBUG
+#define DEBUG_VALIDATE() Scene::DebugValidate(__PRETTY_FUNCTION__)
+#else
+#define DEBUG_VALIDATE() do {} while(0)
```

This is actually not needed. Empty strings will get silently discarded.